_pages/business-units/18f/chapters/engineering.md

```diff
@@ -106,20 +106,16 @@ Most of the Engineering team works on a mix of partner agency projects and inter
 
 ### Tools
 
-[#django](https://gsa-tts.slack.com/messages/django), [#containers](https://gsa-tts.slack.com/messages/containers), [#elk](https://gsa-tts.slack.com/messages/elk), [#git](https://gsa-tts.slack.com/messages/git), [#go](https://gsa-tts.slack.com/messages/go), [#javascript](https://gsa-tts.slack.com/messages/javascript), [#mac](https://gsa-tts.slack.com/messages/mac), [#meteorjs](https://gsa-tts.slack.com/messages/meteorjs), [#python](https://gsa-tts.slack.com/messages/python) (and [#learnpython](https://gsa-tts.slack.com/messages/learnpython)), [#ruby](https://gsa-tts.slack.com/messages/ruby).
+[#django](https://gsa-tts.slack.com/messages/django), [#elk](https://gsa-tts.slack.com/messages/elk), [#go](https://gsa-tts.slack.com/messages/go), [#javascript](https://gsa-tts.slack.com/messages/javascript), [#python](https://gsa-tts.slack.com/messages/python) (and [#learnpython](https://gsa-tts.slack.com/messages/learnpython)), [#ruby](https://gsa-tts.slack.com/messages/ruby).
```

Comment:
All for cleaning up dead links, but maybe the tools are still relevant, just without a slack channel?

Reply:
For sure! I can't speak to them being relevant, but I'm happy to link to the documentation for those tools which no longer have active Slack channels?